obvci/conda_tools/build_directory.py

```diff
@@ -135,15 +152,19 @@ def calculate_existing_distributions(self, recipe_metas):
                                 for meta in recipe_metas])))
         return existing_distributions
 
+    def recipes_to_build(self, recipes):
+        existing_distributions = self.calculate_existing_distributions(recipes)
+        return [recipe in existing_distributions for recipe in recipes]
```

Comment:
Don't think this is doing the right thing. This is the problem with writing changes and *not* pushing them immediately... :disappointed: